src/Style/Colors.ts

```diff
@@ -1,31 +1,47 @@
 export const PrimaryColor = {
+  glintsyellow: '#FFF240',
   glintsred: '#EC272B',
   glintsblue: '#0BAEEC',
-  glintsyellow: '#FFF240',
 };
 
 export const SecondaryColor = {
-  white: '#FFFFFF',
-  whitesmoke: '#F3F3F3',
+  actionblue: '#017EB7',
+  green: '#93BD49',
+  darkgreen: '#008464',
+  magenta: '#FF00E5',
+  violet: '#9013FE',
+  orange: '#F5A623',
+
+  // deprecated
   ivory: '#FFFAEA',
-  black: '#000000',
   lighterblack: '#AAAAAA',
-  lightblack: '#777777',
-  grey: '#777777',
-  darkgrey: '#777777',
-  lightgrey: '#C6C6C6',
   lightergrey: '#EEEEEE',
-  purplepizza: '#FF00E5',
   blue: '#0869C2',
   darkerblue: '#027EB7',
-  bluegem: '#9013FE',
-  actionblue: '#017EB7',
-  green: '#93bd49',
   lightgreen: '#EEF6F4',
-  darkgreen: '#008464',
-  orange: '#F5A623',
   lightorange: '#FFF6E8',
   lightred: '#FFF2F3',
   lightblue: '#D6F4FF',
   darkblue: '#004b68',
+
+  // renamed
+  purplepizza: '#FF00E5',
```

Comment:
oh there two are renamed, but it seems they are duplicated, and in the storybook, I only see `magenta` and `violet`, which two do you want to be used?

Reply:
We will use this:
```
magenta: '#FF00E5',
violet: '#9013FE',
```

I'm leaving `purplepizza` and `bluegem` there until I refactor all the usage of them in our codebases.

Reply:
ok!